apps/api/src/app/workflows-v2/mappers/notification-template-mapper.ts

```diff
@@ -63,10 +63,22 @@ function toMinifiedWorkflowDto(template: NotificationTemplateEntity): WorkflowLi
     updatedAt: template.updatedAt || 'Missing Updated At',
     stepTypeOverviews: template.steps.map(buildStepTypeOverview).filter((stepTypeEnum) => !!stepTypeEnum),
     createdAt: template.createdAt || 'Missing Create At',
-    status: WorkflowStatusEnum.ACTIVE,
+    status: getWorkflowStatus(template),
```

Comment:
i think this one should work as the workflow should be updated.

```suggestion
    status: template.status,
```